cmd/bootstrap/cmd/key.go

```diff
@@ -106,25 +107,32 @@ func keyCmdRun(_ *cobra.Command, _ []string) {
 	}
 }
 
-func generateKeys() (crypto.PrivateKey, crypto.PrivateKey, error) {
+func generateKeys() (crypto.PrivateKey, crypto.PrivateKey, []byte, error) {
 
 	log.Debug().Msg("will generate networking key")
 	networkSeed := validateSeed(flagNetworkSeed)
 	networkKey, err := utils.GenerateNetworkingKey(networkSeed)
 	if err != nil {
-		return nil, nil, fmt.Errorf("could not generate networking key: %w", err)
+		return nil, nil, nil, fmt.Errorf("could not generate networking key: %w", err)
 	}
 	log.Info().Msg("generated networking key")
 
 	log.Debug().Msg("will generate staking key")
 	stakingSeed := validateSeed(flagStakingSeed)
 	stakingKey, err := utils.GenerateStakingKey(stakingSeed)
 	if err != nil {
-		return nil, nil, fmt.Errorf("could not generate staking key: %w", err)
+		return nil, nil, nil, fmt.Errorf("could not generate staking key: %w", err)
 	}
 	log.Info().Msg("generated staking key")
 
-	return networkKey, stakingKey, nil
+	log.Debug().Msg("will generate db encryption key")
+	secretsDBKey, err := utils.GenerateSecretsDBEncryptionKey()
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("could not generate secrets db encryption key: %w", err)
+	}
+	log.Info().Msg("generated db encryption key")
```

Comment:
```suggestion
	log.Info().Msg("saved db encryption key")
```
otherwise, we are printing [the same log](https://github.com/onflow/flow-go/pull/1340/files#diff-06a9e68dbe2d83c5f245036e8a2795aa6b672666053982ec61fd4a3b5a084f0aR49) twice

Reply:
The two logs are for two separate commands. One is for setting up all of your keys at once (`key.go`) the other is only for creating the encryption key (`db_encryption_key.go`), so we wouldn't see these logs twice running either command.